src/commands/stepfunctions/helpers.ts

```diff
@@ -142,6 +143,34 @@ export const shouldUpdateStepForTracesMerging = (step: StepType): boolean => {
   return false
 }
 
+// Truth table
+// Input                    | Expected
+// -------------------------|---------
+// Empty object             | true
+// undefined                | true
+// not object               | false
+// object without CONTEXT.$ | true
+// object with CONTEXT.$    | false
```

Comment:
Why have a truth table when the function is unit tested?

Reply:
Can you have 1 test case per line in this truth table, and remove the truth table?

Reply:
Due to the way Stepfunctions are defined, the Input can be any valid JSON value. We really only care about the case where Input is {"CONTEXT.$":{...}}, so that's how the type is defined. Instead of changing it to any, I'd rather leave a note to future developers explaining what this function returns, and cover it with test cases as best I can.

I added a test case I missed, that was possible to add without changing the type of Input.